lib/mempool.go

```diff
@@ -1786,6 +1786,100 @@ func ComputeTransactionMetadata(txn *MsgDeSoTxn, utxoView *UtxoView, blockHash *
 			QuantityToFillInBaseUnits:                 realTxMeta.QuantityToFillInBaseUnits,
 		}
 
+	case TxnTypeCreateUserAssociation:
+		realTxMeta := txn.TxnMeta.(*CreateUserAssociationMetadata)
+		targetUserPublicKeyBase58Check := PkToString(realTxMeta.TargetUserPublicKey.ToBytes(), utxoView.Params)
+		appPublicKeyBase58Check := PkToString(realTxMeta.AppPublicKey.ToBytes(), utxoView.Params)
+
+		txnMeta.CreateUserAssociationTxindexMetadata = &CreateUserAssociationTxindexMetadata{
+			TargetUserPublicKeyBase58Check: targetUserPublicKeyBase58Check,
+			AppPublicKeyBase58Check:        appPublicKeyBase58Check,
+			AssociationType:                string(realTxMeta.AssociationType),
+			AssociationValue:               string(realTxMeta.AssociationValue),
+		}
+
+		txnMeta.AffectedPublicKeys = append(txnMeta.AffectedPublicKeys, &AffectedPublicKey{
```

Comment:
I'm open to that. But are those affected public keys used primarily for notifications? Because it may be annoying as an app user to get a notification every time someone creates an app-specific association related to your app?